src/Evolution/Systems/Cce/AnalyticSolutions/LinearizedBondiSachs.hpp

```diff
@@ -25,7 +26,74 @@ class ComplexDataVector;
 /// \endcond
 
 namespace Cce {
+
+namespace InitializeJ {
+/// First hypersurface Initialization for the
+/// `Cce::Solutions::LinearizedBondiSachs` analytic solution.
+///
+/// This initialization procedure should not be used except when the
+/// `Cce::Solutions::LinearizedBondiSachs` analytic solution is used.
+struct LinearizedBondiSachs : InitializeJ {
```

Comment:
Could this commit be a follow-up PR? I find it difficult to review a PR that mixes this sort of math-heavy code and the control-flow / "add an executable" code that is the rest of this PR. I think this PR could actually only add the test-executable with one input file that works, and then follow-up PRs can fix the individual analytic solutions. That would give the individual commits the attention they deserve :)